### PR TITLE
Fixed PHP 7.2 error

### PR DESCRIPTION
**Pull request by mickaelandrieu**
You will this error thrown everywhere in PrestaShop BO if you're using PHP 7.2

capture

### CODE REVIEW

File: controllers/admin/AdminGamificationController.php [hunk 1]
```diff
@@ -15,7 +15,7 @@ public function __construct()
         }
     }
     
-    public function setMedia()
+    public function setMedia($isNewTheme = false)
```
Comment (Quetzacoalt91) [inline controllers/admin/AdminGamificationController.php]: Can you please forward it to the parent::setMedia() please?

Comment (mickaelandrieu) [reply controllers/admin/AdminGamificationController.php]: done!

**Comment by Quetzacoalt91** [comment]
Thank you @mickaelandrieu

**Comment by rGaillard** [comment]
This same fix can be done for the 1.6 module version?

**Comment by mickaelandrieu** [comment]
@rGaillard afaik we don't support PHP 7.2 on 1.6

**Pull request merged**